### ISSUE TITLE
Que schedule configuration

### ISSUE THREAD

**Issue by jish** [hlascelles/que-scheduler#3]
The Readme states:

> "The schedule file is a list of que job classes with arguments and a schedule frequency (in crontab syntax). The format is a superset of the resque-scheduler config format, so it they can be used as-is with no modification"

I have a `resque-scheduler` config with:

1. Time zones in the `cron:` definition
2. A key called `every:` instead of `cron:`

```yaml
Foo:
  cron: "0 7 * * * America/Los_Angeles"
```

```yaml
Bar:
  every: "10m"
```

Does `que-scheduler` support this syntax?

**Comment by hlascelles** [comment]
Not yet, I've been using the cron parser [fugit](https://github.com/floraison/fugit#fugitcron).

However, I can [see](https://snippets.aktagon.com/snippets/701-how-to-parse-a-cronline-crontab-cron-with-ruby) that it is almost certainly possible to use`rufus-scheduler` in the same manner.

I've changed the README to reference fugit for now, but will investigate using [rufus-scheduler](https://github.com/jmettraux/rufus-scheduler), as then there will be even greater config file parity with [resque-scheduler](https://github.com/resque/resque-scheduler).

**Comment by hlascelles** [comment]
I have [created a branch](https://github.com/hlascelles/que-scheduler/compare/use-rufus-scheduler-to-parse-cron) that uses rufus-scheduler (and thus allows timezones).

I've also [raised an issue](https://github.com/floraison/fugit/issues/2) with fugit to see what the desired future state should be.

**Comment by hlascelles** [comment]
A [new fugit](https://github.com/floraison/fugit/issues/2#issuecomment-376008274) is coming, and builds are passing: https://github.com/hlascelles/que-scheduler/pull/8

I'll cut a new gem when fugit is released.

**Comment by hlascelles** [comment]
Timezone support seems to be working great. `every` support will be investigated, but is not planned soon.

**Comment by hlascelles** [comment]
Closing via https://github.com/hlascelles/que-scheduler/pull/8 to add timezone support. 

Released as [1.2.0](https://rubygems.org/gems/que-scheduler/versions/1.2.0).

**Comment by jish** [comment]
Thanks!

Josh
